webapp/static/interop-data.json

```diff
@@ -1695,8 +1695,8 @@
         "description": "URLPattern",
         "mdn": "https://developer.mozilla.org/en-US/docs/Web/API/URL_Pattern_API",
         "spec": "https://urlpattern.spec.whatwg.org/",
-        "tests": "/results/urlpattern?label=master&product=chrome&product=edge&product=firefox&product=safari&view=interop&q=label%3Ainterop-2025-urlpattern",
-        "mobile_tests": "/results/urlpattern?label=master&product=chrome&product=edge&product=firefox&product=safari&view=interop&q=label%3Ainterop-2025-urlpattern",
+        "tests": "/results/urlpattern?label=master&product=chrome&product=edge&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2025-urlpattern",
+        "mobile_tests": "/results/urlpattern?label=master&product=chrome_android&product=firefox_android&aligned&view=interop&q=label%3Ainterop-2025-urlpattern",
```

Comment:
Good catch with `chrome`/`firefox`->`chrome_android`/`firefox_android` for this file and interop-data.js!